src/client/mod.rs

```diff
@@ -207,6 +207,51 @@ impl GaloyClient {
         }
     }
 
+    pub fn intraleger_usd_send(
+        &self,
+        username: String,
+        amount: Decimal,
+        memo: Option<String>,
+    ) -> anyhow::Result<UsdPaymentSendResult> {
+        let me = self.me()?;
+        let wallet_id = me.default_account.default_wallet_id;
+
+        let recipient_wallet_id = self.default_wallet(username)?;
```

Comment:
What if the default wallets of the sender and receiver are different - one USD and other BTC?  Will `IntraLedgerUsdPaymentSend` work in that case?
Last time I tried this, it led to a `MismatchCurrency` error.

Reply:
Yes, it's supposed to work. Will test on staging.

Reply:
Fixed